_extras/sequence_alignment.md

```diff
@@ -0,0 +1,516 @@
+Since the command has become quite long, we can use the continuation symbol `\` to let the shell know that the file is written on 2 lines rather than a single, long line. This time we can align different sequences, for example: GIP.fa and GLP-2.fa.
+
+~~~
+$ docker run --rm --mount type=bind,source=${PWD},target=/data biocontainers/emboss:v6.6.0dfsg-7b1-deb_cv1  \
+needle GIP.fa GLP-2.fa -outfil gip-glp2.needle -gapopen 10 -gapextend 0.5
+~~~
+{: .language-bash}
+~~~
+ Needleman-Wunsch global alignment of two sequences 
+~~~
+{: .output}
+
+We can then look just at the bottom of the resulting alignment file:
+
+~~~
+$ tail gip-glp2.needle 
+~~~
+{: .language-bash}
+~~~
+#
+#=======================================
+
+GIP                1 YAEGTFISDYSIAMDKIRQQDFVNWLLAQ----     29
+                     :|:|:|..:.:..:|.:..:||:|||:..    
+GLP-2              1 HADGSFSDEMNTILDNLAARDFINWLIQTKITD     33
+
+#---------------------------------------
+#---------------------------------------
+~~~
+{: .output}
+
+This can easily be turned into a loop, for example aligning all sequences to that of glucagon.
+The filename extension `.fa` is removed with the `bash` method using `basename` so that the resulting file name only has `.needle` as an extension.  
+Some of the loop commands, ending with `;` can be written on separate lines.
+However, due to the loop nature of the command, it is not possible to use the continuation symbol `\` as we just did previously.
+
+~~~
+$ for f in G*.fa; do  b=`basename $f .fa`;    
+docker run --rm --mount type=bind,source=${PWD},target=/data biocontainers/emboss:v6.6.0dfsg-7b1-deb_cv1 needle glucagon.fa $f -outfil gluc_$b.needle -gapopen 10 -gapextend 0.5; 
+done
+~~~
+{: .language-bash}
+~~~
+Needleman-Wunsch global alignment of two sequences
+Needleman-Wunsch global alignment of two sequences
+Needleman-Wunsch global alignment of two sequences
+~~~
+{: .output}
+
+The output shows that the program ran 3 times, as expected. The resulting files can be found within the current directory that you can explore using the `cat` or `tail` shell commands for example.
+
+> ## Multiple sequence alignment
+> EMBOSS only provides a "wrapper" ([emma](http://emboss.sourceforge.net/apps/release/6.6/emboss/apps/emma.html)) calling an external multiple sequence 
+> alignment named `clustalw` which is an older algorithm. For protein sequences one of the recommended algorithm is a similar, newer algorithm named `clustalomega` 
+> that we will use within a different docker image.   
+> From web [clustalomega documentation](https://www.ebi.ac.uk/seqdb/confluence/display/THD/Clustal+Omega): 
+> *Clustal Omega is a multiple sequence alignment program for aligning three or more sequences together in a computationally efficient and accurate manner. 
+> It produces biologically meaningful multiple sequence alignments of divergent sequences.*
+> (See Sievers and Higgins (2018).)
+> 
+{: .callout}
```

Comment:
I don't agree on this, and perhaps it may be that I don't know how to make it appear clearly as a new chapter: we are going to be doing something new, not possible with the EMBOSS image, and we are going to pull a new image, for a new program "clustalomega". If there is a better way to make this a clear distinction with previous let me know... I leave it as it is now.